doc/source/examples.rst

```diff
@@ -247,6 +247,56 @@ In that case, if a pull request gets labelled with ``work-in-progress``, it
 won't be merged, even if approved by 2 contributors and having Travis CI
 passing.
 
+⚡️ Using Labels to Prioritize Merge
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When ``smart`` :ref:`strict merge` is enabled and you have many pull request
+waiting to be merge. some of them are perhaps more urgent. In that case, you
+can add a condition using a `label
+<https://help.github.com/articles/labeling-issues-and-pull-requests/>`_ and
+configure the priority option of :ref:`merge action`:
+
+.. code-block:: yaml
+
+    pull_request_rules:
+      - name: automatic merge of hotfix 🚑 (high priority)
+        conditions:
+          - status-success=Travis CI - Pull Request
+          - "#approved-reviews-by>=2"
+          - base=master
+          - label=hotfix 🚑
+        actions:
+          merge:
+            method: merge
+            strict: smart
+            priority: high
+      - name: automatic merge of bot 🤖 (low priority)
+        conditions:
+          - author~=^dependabot(|-preview)\[bot\]$
+          - status-success=Travis CI - Pull Request
+          - "#approved-reviews-by>=2"
+          - base=master
+        actions:
+          merge:
+            method: merge
+            strict: smart
+            priority: low
+      - name: automatic merge for master when reviewed and CI passes
+        conditions:
+          - status-success=Travis CI - Pull Request
+          - "#approved-reviews-by>=2"
+          - base=master
```

Comment:
do you need to have `-label=hotfix 🚑`?
To me, the last rule is going to be used and therefore it's going to apply therefore the priority will be set to `medium`.

Reply:
It's the first that match that merge the PR.

Reply:
Hm, it's going to put it in the queue, not merge right way… which makes me think as a user it's not clear that it's going to stop there or not.